.gitignore

```diff
@@ -19,6 +19,7 @@
 !/src/themes/PrinterTheme
 !/src/themes/RssTheme
 !/src/themes/SeaBreezeTheme
+!/src/themes/MockTheme
```

Comment:
it appears that this branch/PR has some old commits that were removed from the git history of the core. rebasing _may_ remove these but if not, you will have to do something different. both commits related to the MockTheme solution must be removed.